source/ios/Additions.cs

```diff
@@ -72,14 +72,17 @@ static void ConvertToNsExceptionAndAbort(object e)
 			var msg = e.ToString();
 
 			var ex = e as Exception;
-			if(ex != null) 
-				name = string.Format("{0}: {1}", ex.GetType().FullName, ex.Message);
-
+			if (ex != null) {
+				name = ex.GetType ().FullName;
+				if (ex.StackTrace != null) {
+					msg = msg.Insert (msg.IndexOf('\n'), "Xamarin Exception Stack:");
```

Comment:
Ah, I was just curious since it packs the info message reason as

`...This is a basic exception.Xamarin Exception Stack:...`

versus the example:

```
...
zero.'

Xamarin Exception Stack:
...
```

If this doesn't matter and there will be front end logic to display that, then this will be fine.